packages/core/src/transports/types.ts

```diff
@@ -1,4 +1,5 @@
 import type { APIEvent, ExceptionEvent, LogEvent, MeasurementEvent, TraceEvent } from '../api';
+import type { EventEvent } from '../api/events';
```

Comment:
This should be imported from `api` directly rather than `api/events`

Reply:
done